lib/solidus_static_content/route_matcher.rb

```diff
@@ -0,0 +1,13 @@
+module SolidusStaticMatcher
```

Comment:
File is under `lib/solidus_static_content`, so shouldn't this be `SolidusStaticContent`?

Reply:
🤦‍♂️😄 Yes.